builder/config_reader.go

```diff
@@ -30,6 +31,23 @@ type BuildpackConfig struct {
 	dist.ImageOrURI
 }
 
+type Assets struct {
+	Packages []Package `toml:"package"`
+}
+
+type Package struct {
+	AssetImageOrURI
```

Comment:
Do we need a dedicated `AssetImageOrURI`, or would `dist.ImageOrUri` work?

Reply:
Ya I made a new object here because `dist.ImageOrURI` has a `buildpackURI` object inside of it. Which reads awkwardly when using assets.